src/autoscaler/pruner/config/config.go

```diff
@@ -23,17 +23,34 @@ var defaultLoggingConfig = LoggingConfig{
 }
 
 type DbConfig struct {
-	MetricsDbUrl string `yaml:"metrics_db_url"`
+	MetricsDbUrl    string `yaml:"metrics_db_url"`
+	AppMetricsDbUrl string `yaml:"app_metrics_db_url"`
 }
 
 type PrunerConfig struct {
```

Comment:
The config seems confused. Why is the DB Url separate from the config of the pruner?
Why isn't there a common config for each?